PowerEditor/src/lastRecentFileList.h

```diff
@@ -44,6 +44,8 @@ class LastRecentFileList
 public:
 	LastRecentFileList() {
 		_userMax = (NppParameters::getInstance())->getNbMaxRecentFile();
+		for (int i = 0; i < NB_MAX_LRF_FILE; i++)
+			_idFreeArray[i] = false;
 	};

```

Comment:
OK, thank you for your info.
The future release will be compiled by VS2015, as well in appveyor.